### DISCUSSION TITLE
Is  spatie/emoji dependable from other packages/fonts/Browsers of the OS?

### DISCUSSION THREAD

**Discussion by sergeynilov** [spatie/emoji#52] [Q&A] [answered]
Hello,
I installed spatie/emoji into my laravel 8 app on my local Kubuntu 18 and wonder if these emojis 
are dependable from other packages/fonts/Browsers of the OS? 

I found a hint that I need  fonts-noto-color-emoji to be installed in the OS...

Thanks!

**Comment by Vusys** [comment, marked as answer]
Since this package prints the native unicode for emoji, support depends on your OS and browser. That varies between OSs and even different browsers on the same OS.

As an example, I'm currently running macOS Mojave which does not support newer emojis like [Face Exhaling](https://emojipedia.org/face-exhaling/), and I see the underlying emoji and not the joined emoji:

![Screenshot 2021-05-11 at 12 36 48](https://user-images.githubusercontent.com/4213522/117809110-a713f780-b255-11eb-88df-e6346eeede35.png)

Someone on Big Sur or iOS 14.5 should see joined emoji as they have a newer emoji font installed. 

According to [Gitlab](https://about.gitlab.com/blog/2018/05/30/journey-in-native-unicode-emoji/), "most Linux environments do not have Unicode emoji support unless you manually install a font", which probably explains your issue.
